i18n/SigDigger_zh_CN.ts

```diff
@@ -2843,4 +2843,645 @@
+        <source>dB</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../ui/GainSlider.ui" line="59"/>
+        <source>NN</source>
+        <translation type="unfinished"></translation>
+    </message>
+  </context>
+  <context>
+    <name>Inspector</name>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="14"/>
+        <source>Form</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="43"/>
+        <source>Channel properties</source>
+        <translation type="finished">信道属性</translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="130"/>
+        <source>Bandwidth (Hz)</source>
+        <translation type="finished">带宽(Hz)</translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="140"/>
+        <source>Doppler correction</source>
+        <translation type="finished">多普勒修正</translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="166"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="178"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="197"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="216"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="235"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="263"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="275"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="301"/>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="836"/>
+        <source>N / A</source>
+        <translation type="finished">不可用</translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="185"/>
+        <source>Satellite</source>
+        <translation type="finished">卫星</translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="204"/>
+        <source>Elevation</source>
+        <translation type="finished">海拔</translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="223"/>
+        <source>Visible</source>
+        <translation type="finished">可见</translation>
+    </message>
+    <message>
+        <location filename="../Default/GenericInspector/GenericInspector.ui" line="242"/>
+        <source>Airmass</source>
+        <translation type="finished">气团</translation>
```

Comment:
I believe this is a literal translation when it should have been a technical term (大气质量, perhaps? See https://zh.wikipedia.org/wiki/%E5%A4%A7%E6%B0%94%E8%B4%A8%E9%87%8F_(%E5%A4%AA%E9%98%B3%E8%83%BD) )

The idea of the air mass is that as you look closer to the horizon, the apparent thickness of atmosphere you observe is bigger, as light (radio waves) arrives in diagonal. It is a number that goes from 1 (if you look right above your head) to infinity (if you look right to the horizon). Of course you will never have infinity air mass in practice, as it would mean the Earth had infinite radius (which we know it hasn't).

Reply:
Excuse my ignorance.

Reply:
No worries! These discussions are part of the translation process. There's a bunch of mixed, field-specific terminology here, you are not supposed to know everything about them without further context.